### PR TITLE
show error message

### PR DESCRIPTION
**Pull request by Deepak101094**
show error message,when response is false..!

### CODE REVIEW

File: src/components/MovieSearch.js [hunk 1]
```diff
@@ -42,12 +42,12 @@ function MovieSearch() {
...
           })
           .then((res) => {
-            // console.log(res.data);
+            // console.log(res);
             setResources(res.data);
-          })
-          .catch((err) => {
-            setError(err);
-            //console.log(err)
+            if(res.data.Error) {
```
Comment (shubuu97) [inline src/components/MovieSearch.js]: use optional chaining like this:

if(res?.data?.Error) {

File: src/components/MovieSearch.js [hunk 2]
```diff
@@ -42,12 +42,12 @@ function MovieSearch() {
...
+              setError("movie not fonnd...!");
```
Comment (shubuu97) [inline src/components/MovieSearch.js]: Capitalise the error message

**Review by shubuu97 — COMMENTED** [review]
Good work 👍 , some small changes

File: src/components/MovieSearch.js (hunk 1 above)
Comment (Deepak101094) [reply src/components/MovieSearch.js]: done

File: src/components/MovieSearch.js (hunk 2 above)
Comment (Deepak101094) [reply src/components/MovieSearch.js]: done

**Pull request merged**